daemon/mgr/network.go

```diff
@@ -411,15 +410,15 @@ func (nm *NetworkManager) getNetworkSandbox(id string) libnetwork.Sandbox {
 
 func initNetworkLog(cfg *config.Config) {
 	if cfg.Debug {
-		netlog.SetLevel(netlog.DebugLevel)
+		logrus.SetLevel(logrus.DebugLevel)
```

Comment:
Libnetwork's logrus version is different from pouchd, so we need to set libnetwork's logrus addintionly.

Reply:
known